.storybook/themes/commerce.scss

```diff
@@ -3,43 +3,543 @@
+					color: var(--global-neutral-7),
+				),
+				button-default: (
+					color: var(--primary-color),
+					background-color: var(--global-white),
+				),
+				button-default-hover: (
+					background-color: var(--global-neutral-1),
+				),
+				carousel: (
+					--slides: 1,
+				),
+				carousel-actions: (
+					display: none,
+					padding-left: var(--global-spacing-4),
+					padding-right: var(--global-spacing-4),
+					//
+					// The width should place the action buttons near the edge of the carousel responsively
+					//
+					// width is derived from adding
+					//   --layout-content-max-width
+					//   2x button widths (--global-font-size-9)
+					//   2x button margins (--global-font-size-5)
+					// then dividing those by
+					//   --layout-max-width
+					// using the formula: (48 + 32 + 1216 + 32 + 48) / 1440
+					//
+					width: calc(calc(var(--layout-content-max-width) + 160) / var(--layout-max-width) * 100%),
+				),
+				carousel-button: (
+					box-shadow: var(--global-box-shadow-1),
+					border-radius: var(--global-spacing-1),
+					background-color: var(--global-white),
+					height: var(--global-spacing-8),
+					width: var(--global-spacing-8),
+				),
+				carousel-track: (
+					gap: var(--global-spacing-2),
+					width: var(--layout-content-scale-width),
+				),
+				date: (
+					font-family: var(--secondary-font-family),
+					font-size: var(--global-font-size-3),
+					line-height: var(--global-line-height-3),
+				),
+				heading: (
+					font-weight: var(--global-font-weight-7),
+				),
+				icon: (
+					fill: var(--global-neutral-8),
+					height: var(--global-spacing-5),
+					width: var(--global-spacing-5),
+				),
+				input: (
+					align-items: flex-start,
+					display: flex,
+					flex-direction: column,
+					gap: var(--global-spacing-1),
+				),
+				input-input: (
+					border: 1px solid var(--global-neutral-5),
+					border-radius: var(--global-border-radius-1),
+					padding: var(--global-spacing-2) var(--global-spacing-3),
+				),
+				input-input-focus: (
+					outline: double 2px var(--global-neutral-5),
+				),
+				input-label: (
+					font-weight: var(--global-font-weight-7),
+				),
+				input-error-input: (
+					border-color: var(--global-red-4),
+				),
+				input-error-input-focus: (
+					outline-color: var(--global-red-4),
+				),
+				input-success-input: (
+					border-color: var(--global-green-5),
+				),
+				input-success-input-focus: (
+					outline-color: var(--global-green-5),
+				),
+				input-warning-input: (
+					border-color: var(--global-orange-2),
+				),
+				input-warning-input-focus: (
+					outline-color: var(--global-orange-2),
+				),
+				link: (
+					color: var(--primary-color),
+				),
+				media-item-fig-caption: (
+					color: var(--global-neutral-5),
+					font-size: var(--global-font-size-2),
+					line-height: var(--global-line-height-2),
+				),
+				pill: (
+					background-color: var(--primary-color),
+					border-radius: 2em,
```

Comment:
this font-weight should be bold 70
![Screen Shot 2022-05-02 at 12 53 15](https://user-images.githubusercontent.com/5950956/166299108-c539520a-956b-4ccf-92df-855b22491209.png)
0

Reply:
I've updated this @ecarlisle

Reply:
We should hold on this, until we get full clarification and alignment on the designs - as the component in this repository is aligned to the Tag component, we currently do not have a badge component

Reply:
@matthewroach : The styles match the Badge component in Figma if that's the match we're looking for. Thanks!

Reply:
Figma has been updated and the Pill is now correctly labelled to the one I original linked you too

Reply:
`font-weight` should be `400`